### PR TITLE
Split place page chart config json file by category

### PR DESCRIPTION
**Pull request by shifucun**
This is to make it easy to 
- maintain the config 
- pull configs that are only relevant to a deployment
- sort the categories

After this is checked in. Need to have a script to sync the concatenated json in g3

### CODE REVIEW

File: server/__init__.py [hunk 1]
```diff
@@ -32,6 +32,17 @@
 
 propagator = google_cloud_format.GoogleCloudFormatPropagator()
 
+RANKED_CATEGORIES = [
```
Comment (beets) [inline server/__init__.py:35]: nit: maybe ORDERED_PLACE_EXPLORER_CATEGORIES or just PLACE_EXPLORER_CATEGORIES

File: static/js/shared/util.ts [hunk 2]
```diff
@@ -14,6 +14,17 @@
  * limitations under the License.
  */
 
+export const topics = [
```
Comment (beets) [inline static/js/shared/util.ts:17]: add a note that this needs to stay in sync with server/__init__.py?

File: server/__init__.py (hunk 1 above)
Comment (beets) [reply server/__init__.py:35]: also add a note that this needs to stay in sync with util.ts?

**Review by beets — APPROVED** [review]
thx for splitting it up!

File: static/js/shared/util.ts (hunk 2 above)
Comment (shifucun) [reply static/js/shared/util.ts]: Done

**Comment by shifucun** [comment]
Thanks for review!

**Pull request merged**